packages/cashier/src/modules/cashier-onboarding/components/cashier-onboarding-side-notes/cashier-onboarding-side-note-fiat.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { getCurrencyDisplayCode } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+import { SideNoteCard } from '../../../../components/side-note-card';
+
+const CashierOnboardingSideNoteFiat: React.FC = observer(() => {
+    const { client, ui } = useStore();
+    const { currency, loginid, is_eu, is_low_risk } = client;
+    const { is_mobile } = ui;
+    const currency_code = getCurrencyDisplayCode(currency);
+    const regulation_text = is_eu ? 'EU' : 'non-EU';
+
+    return (
+        <SideNoteCard
+            title={
+                is_low_risk
+                    ? localize('This is your {{regulation_text}} {{currency_code}} account {{loginid}}', {
+                          regulation_text,
+                          currency_code,
+                          loginid,
+                      })
+                    : localize('This is your {{currency_code}} account {{loginid}}', { currency_code, loginid })
+            }
+            hide_paddings={!is_mobile}
+            description={
+                <Localize
+                    i18n_default_text='If you want to change your account currency, please contact us via <0>live chat</0>.'
+                    components={[
```

Comment:
Is there any way to achieve the same thing using the `localize` function instead of the `Localize` component so I can keep the type as `string` instead of making it `JSX.Element`? 🤔